postman/collections/denner-api.postman-v2.json

```diff
@@ -2190,6 +2160,184 @@
 					"response": []
 				},
 				{
+					"name": "/print-publications/{publication_id}/links",
+					"request": {
+						"url": "{{advertising_url_base}}/print-publications/{{publication_id}}/links",
+						"method": "GET",
+						"header": [
+							{
+								"key": "Accept",
+								"value": "{{header_accept}}",
+								"description": ""
+							},
+							{
+								"key": "App-ID",
+								"value": "{{advertising_header_app_id}}",
+								"description": ""
+							},
+							{
+								"key": "App-Key",
+								"value": "{{advertising_header_app_key}}",
+								"description": ""
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"description": "List links of a print publication"
+					},
+					"response": []
+				},
+				{
+					"name": "/print-publications/{publication_id}/links/{link_id}",
+					"request": {
+						"url": "{{advertising_url_base}}/print-publications/{{publication_id}}/links/{{link_id}}",
+						"method": "GET",
+						"header": [
+							{
+								"key": "Accept",
+								"value": "{{header_accept}}",
+								"description": ""
+							},
+							{
+								"key": "App-ID",
+								"value": "{{advertising_header_app_id}}",
+								"description": ""
+							},
+							{
+								"key": "App-Key",
+								"value": "{{advertising_header_app_key}}",
+								"description": ""
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"description": "List a single link (Link) for a print publication (Print-Werbemittel)"
+					},
+					"response": []
+				},
+				{
+					"name": "/print-publications/{publication_id}/links",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"type": "text/javascript",
+								"exec": [
+									"// Save created publication article's id in environment",
+									"var jsonData = JSON.parse(responseBody),",
+									"    id = (jsonData) ? jsonData.id : null;",
+									"    ",
+									"postman.setEnvironmentVariable(\"publication_article_id\", id);",
+									"",
+									"tests[\"Setup new publication_article_id in environment\"] = !!id;",
+									""
+								]
+							}
+						}
+					],
```

Comment:
This script does not work for links (if anything should be saved to an environment var, it's a link).